### PR TITLE
Add stand mode for local planner, remove twist body planner

### PR DESCRIPTION
**Pull request by jcnorby**

### CODE REVIEW

**Comment by yanhaoy** [comment]
Did you adjust the line width of the formatter? Why are there so many newlines...

File: local_planner/src/local_planner.cpp [hunk 1]
```diff
@@ -396,32 +408,38 @@ void LocalPlanner::getStateAndTwistInput() {
...
+  for (int i = 0; i < num_feet_; i++) {
+    support_center.x() +=
+        robot_state_msg_->feet.feet[i].position.x / ((double)num_feet_);
+    support_center.y() +=
+        robot_state_msg_->feet.feet[i].position.y / ((double)num_feet_);
+  }
+
+  // Walk if velocity commanded, current velocity exceeds threshold, or too far
+  // from center of support
+  if (cmd_vel_.norm() > 1e-1 || current_state_.segment(6, 2).norm() > 1e-1 ||
```
Comment (ardalantj) [inline local_planner/src/local_planner.cpp:423]: Could make each of these conditions a lambda function for better readability

**Comment by ardalantj** [comment]
Looks good besides the lambda function comment.

**Review by ardalantj — APPROVED** [review]

**Comment by justinyim** [comment]
Add ros-melodic-teleop-twist-joy to setup_deps since the joystick does not work on a fresh system without it.

File: local_planner/src/local_planner.cpp (hunk 1 above)
Comment (jcnorby) [reply local_planner/src/local_planner.cpp]: Used a bool for this.

File: local_planner/src/local_planner.cpp [hunk 2]
```diff
@@ -341,6 +354,11 @@ void LocalPlanner::getStateAndReferencePlan() {
   body_plan_.row(0) = current_state_;
   foot_positions_body_.row(0) = current_foot_positions_body_;
   foot_positions_world_.row(0) = current_foot_positions_world_;
+
+  // Stand if the plan has been tracked
+  if ((current_state_ - ref_body_plan_.bottomRows(1)).norm() <= 5e-2) {
```
Comment (ardalantj) [inline local_planner/src/local_planner.cpp:359]: Does this need to be a constant?

Comment (jcnorby) [reply local_planner/src/local_planner.cpp]: Updated.

**Review by ardalantj — APPROVED** [review]

**Pull request merged**